src/containers/Logout/LogoutFederated.tsx

```diff
@@ -5,21 +5,23 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import React from 'react';
+import { useEffect } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { actions } from '../../modules/session/session';
 
-export class LogoutFederated extends React.Component {
-  componentDidMount() {
-    const { logout } = this.props;
+type Logout = { federated: boolean; returnToLogin?: boolean };
+interface Props {
+  logout: (arg0: Logout) => void;
+}
+
+export const LogoutFederated = ({ logout }: Props) => {
+  useEffect(() => {
     logout({ federated: true });
-  }
+  });
 
-  render() {
-    return null;
-  }
-}
+  return null;
+};
```

Comment:
Siden du gjør om en `componentDidMount` så burde du ha en tom dependency array i useEffect

Reply:
Dette er vel et identisk tilfelle som det jeg nevnte i kommentaren min på LogoutSession.tsx. For at det skal fungere må eslint-regelen disables.